tests/python_functional/src/syslog_ng/syslog_ng_paths.py

```diff
@@ -53,6 +53,7 @@ def set_syslog_ng_paths(self, instance_name):
                 "stdout": Path(working_dir, "syslog_ng_{}_stdout".format(instance_name)),
             },
             "binary_file_paths": {
+                "syslog_ng_tools": Path(install_dir, "bin"),
```

Comment:
I would use separate instances for `slogkey`, and `slogverify` binaries, as we did it before with `syslog-ng` and `syslog-ng-ctl`. The `bin/` dir will contain more binaries (loggen, persist-tool, dqtool, ...) than `sbin/` dir, and we would like to call them also directly not through an intermediate level (syslog-ng-tools).

Reply:
Fixed.